examples/gpt2/README.md

```diff
@@ -6,10 +6,10 @@ OpenAI GPT-2 model was proposed in [Language Models are Unsupervised Multitask L
 
 - Speed on single NVIDIA-V100-16GB
 
-  |       BatchSize       |        64       |      128       |
-  |:---------------------:|:---------------:|:--------------:|
-  |   transformers_v3.0.2 |   3.0 samples/s |      OOM       |
-  |   above + fastseq     |  11.2 samples/s | 16.5 samples/s |
+  |       BatchSize       |        64       |      128       |      256       |
+  |:---------------------:|:---------------:|:--------------:|:--------------:|
+  |   transformers_v4.12.0 |   3.9 samples/s |      3.8 samples/s       |      OOM       |
```

Comment:
for transformer, BatchSize = 128 is slower than 64. Did we double check it?

Reply:
Yes. Running each 3x results in: 3.9 3.9 3.8 for bsz=64 and 3.8 3.8 3.9 for bsz=128